docs/TASKS.md

```diff
@@ -114,21 +114,52 @@ An example json task that uses remote targets can be found under [examples](http
 
 #### collect
 
-The collect section describes which metrics to collect. Metrics can be enumerated explicitly via:
- - a concrete _namespace_
- - a wildcard, `*`
- - a tuple, `(m1|m2|m3)`
+The collect section describes which metrics are requested to be collected.
 
-The tuple begins and ends with brackets and items inside are separeted by vertical bar. It works like logical `or`, so it gives an error only if none of these metrics can be collected.
+Metrics can be enumerated using:
 
-| Metrics declared in task manifest | Collected metrics                                              |
-|:----------------------------------|:---------------------------------------------------------------|
-| /intel/mock/\*                    | /intel/mock/foo <br/> /intel/mock/bar <br/> /intel/mock/\*/baz |
-| /intel/mock/(foo\|bar)            | /intel/mock/foo <br/> /intel/mock/bar <br/>                    |
-| /intel/mock/\*/baz                | /intel/mock/\*/baz                                             |
+ a) **concrete _namespace_**
```

Comment:
I think it's not so easy - these points are in "collect section", so it needs to be `##### Concrete namespace`, so it will be really small (as you can see below)
##### Concrete namespace

Based on that, I prefer to keep that in this form as it is. Do You agree?